modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb

```diff
@@ -454,6 +465,17 @@ class Form526ResponseSwagger
                         key :description, 'How the veteran got the disability.'
                         key :example, 'Caused by a service-connected disability\\nLengthy description'
                       end
+
+                      property :specialIssues do
```

Comment:
@bastosmichael I'm not positive if `specialIssues` needs to be in `secondaryDisabilities` or not.

Reply:
I do not believe it does, not sure where you got that assumption?

Reply:
When reviewing:

https://github.com/department-of-veterans-affairs/vets-json-schema/blob/0f7d93fdcff0139345998d95a80c2cfd1cb65f7a/src/schemas/21-526EZ-allclaims/schema.js#L125

Line 29 `specialIssues` is nested under `disabilitiesBaseDef`. And in turn `disabilitiesBaseDef` is referenced on line 149 and 153 for both `disabilities` and `secondaryDisabilities`.

Reply:
alright it looks like you did your homework, good job.